tasks.py

```diff
@@ -0,0 +1,96 @@
+"""
+PyInvoke developer task file
+
+These tasks can be run using `invoke <NAME>` or `inv <NAME>` from the project root.
+
+To show all available tasks `invoke --list`
+
+To show task help page `invoke <NAME> --help`
+"""
+import invoke
+
+from scripts import check_type_hint_coverage
+
+
+@invoke.task
+def sort(ctx, path=".", check=False, exclude=None):
+    """Sort module imports"""
+    args = ["isort", path, "--profile", "black"]
+    if check:
+        args.append("--check-only")
+    if exclude:
+        args.extend(["--skip", exclude])
+    ctx.run(" ".join(args))
+
+
+@invoke.task
+def fmt(ctx, path=".", sort_=True, check=False, exclude=None):
+    """Run code formatter"""
+    if sort_:
+        sort(ctx, path, check=check, exclude=exclude)
+
+    args = ["black", path]
+    if check:
+        args.append("--check")
+    if exclude:
+        args.extend(["--exclude", exclude])
+    ctx.run(" ".join(args))
+
+
+@invoke.task
+def lint(ctx, path="great_expectations/core"):
+    """Run code linter"""
+    ctx.run(" ".join(["flake8", path]))
+
+
+@invoke.task
+def hooks(ctx, changes_only=False, diff=False):
+    """Run pre-commit hooks"""
+    cmds = ["pre-commit", "run"]
+    if diff:
+        cmds.append("--show-diff-on-failure")
+    # used in CI - runs faster and only checks files that have changed
+    if changes_only:
+        cmds.extend(["--from-ref", "origin/HEAD", "--to-ref", "HEAD"])
+    else:
+        cmds.extend(["--all-files"])
+    ctx.run(" ".join(cmds))
+
+
+@invoke.task(iterable=["modules"])
+def types(ctx, modules, coverage=False, install_types=False):
```

Comment:
Might change this to `type_check`, which when then be called by `invoke type-check`.
It feels more natural but it's more typing 😆 than `invoke types`